source/Xunit.Gherkin.Quick.UnitTests/GherkinScenarioOutlineTests.cs

```diff
@@ -307,12 +307,578 @@ public void ApplyExampleRow_Digests_Row_Values_Into_Scenario_With_DocString_In_S
+                    {
+                        new TableRow(null, new[] { new TableCell(null, "Column") }),
+                        new TableRow(null, new[] { new TableCell(null, parametrizeDataTable ? inputSentence : normalText)}),
+                    })),
+                    new Gherkin.Ast.Step(null, "Given", normalText, new DocString(
+                        null, "type", parametrizeDocString ? inputSentence : normalText)),
+                },
+                new Gherkin.Ast.Examples[]
+                {
+                    new Gherkin.Ast.Examples(
+                        null,
+                        null,
+                        null,
+                        "existing example",
+                        null,
+                        new Gherkin.Ast.TableRow(null, new Gherkin.Ast.TableCell[]
+                        {
+                            new Gherkin.Ast.TableCell(null, example)
+                        }),
+                        new Gherkin.Ast.TableRow[]
+                        {
+                            new Gherkin.Ast.TableRow(null, new Gherkin.Ast.TableCell[]
+                            {
+                                new Gherkin.Ast.TableCell(null, "1")
+                            })
+                        })
+                });
+
+            //act / assert.
+            var exception = Assert.Throws<InvalidOperationException>(() => sut.ApplyExampleRow("existing example", 0));
+            Assert.Contains("Examples table did not provide value for `a`", exception.Message);
+        }
+
+        [Theory]
+        [InlineData(" initiallyspaced", "value is < initiallyspaced>")]
+        [InlineData("spacedinthened ", "value is <spacedinthened >")]
+        [InlineData(" spaced everywhere ", "value is < spaced everywhere >")]
+        [InlineData("^initiallywrong", "value is <^initiallywrong>")]
+        [InlineData(">initiallywrong", "value is <>initiallywrong>")]
+        [InlineData("!initiallywrong", "value is <!initiallywrong>")]
+        [InlineData("wronginthened^", "value is <wronginthened^>")]
+        [InlineData("wronginthened<", "value is <wronginthened<>")]
+        [InlineData("wronginthened!", "value is <wronginthened!>")]
+        [InlineData("!wrong everywhere!", "value is <!wrong everywhere!>")]
+        [InlineData(">wrong everywhere<", "value is <>!wrong everywhere!<>")]
+        [InlineData("", "value is <>")]
+        [InlineData(" ", "value is < >")]
+        [InlineData("  ", "value is <  >")]
+        [InlineData("   ", "value is <   >")]
+        [InlineData("<", "value is <<>")]
+        [InlineData(">", "value is <>>")]
+        [InlineData("^", "value is <^>")]
+        [InlineData("a<", "value is <a<>")]
+        [InlineData("a^", "value is <a^>")]
+        [InlineData("a<>a", "value is <a<>a>")]
+        [InlineData("a<|^|>a", "value is <a<|^|>a>")]
+        public void DontApplyExampleRow_Starts_With_Invalid_Character(string example, string inputSentence)
```

Comment:
Cases when the parser doesn't see a sequence as a placeholder for an example, so it is pasted as regular text (even if there is an Example with this name! So you could be interested to change it in the future -- I don't see it as a problem for now).